### ISSUE TITLE
[BUG] Training Fails - GK700RBii

### ISSUE THREAD

**Issue by jtemple967** [sdatkinson/neural-amp-modeler#449]
Checking that we have all of the required audio files...
Found v3_0_0.wav, version 3.0.0
Delays:
 Blip  1: 67
 Blip  2: 67
After aplying safety factor of 4, the final delay is 63
Plotting the latency for manual inspection...

Set latency to recommended 63
V3 checks...
Replicate ESR is 2.01217866.
Validation replicates have a self-ESR of over 0.01. Your gear doesn't sound like itself when played twice!

![image](https://github.com/user-attachments/assets/bf33ada0-f10b-409d-87fc-4dd44ee3f2f8)
[output.zip](https://github.com/user-attachments/files/16458702/output.zip)

Possible causes:
 * Your signal chain is too noisy.
 * There's a time-based effect (chorus, delay, reverb) turned on.
 * Some knob got moved while reamping.
 * You started reamping before the amp had time to warm up fully.
Failed checks!
(To disable this check, run AT YOUR OWN RISK with `ignore_checks=True`.)
Exiting core training...
No model returned; skip exporting!

**Comment by vossenv** [comment]
Looks like your output.wav is slightly longer than v3 - or at least, the one in your zip file is.  I tried shortening it up and it seems to be working (ESR 0.001):

https://drive.google.com/file/d/1WE-mPt1XHalcksKQBYznNkJ5s47j8jFn/view?usp=sharing

**Comment by jtemple967** [comment]
@vossenv can you send me the edited WAV file? I made double sure they are the same length and still get the ESR error.

**Comment by vossenv** [comment]
@jtemple967  sure:

https://drive.google.com/file/d/1J0Y9XjLuH6fN7Y3jqL33Hg5_HlaxjF90/view?usp=drive_link

**Comment by jtemple967** [comment]
@vossenv okay, then with your help I'm making progress. I figured out how I was exporting a longer file in my DAW and through the export process.  I can't quite get my file length to match yours exactly but I have gotten the ESR down to just over .01. It still fails but the ESR is much better. I will continue tweaking the exported file.

**Comment by jtemple967** [comment]
Okay, back at it trying to get this to train. So after seeing most people using Reaper I downloaded Reaper and did the capture/export with that tool. The sizes of the files match exactly.  However...the ESR is still over 2.0. I'm just scratching my head here trying to figure out what I'm doing wrong.

@vossenv would you mind sharing what settings you use when you Render your WAV files in Reaper?

**Comment by jtemple967** [comment]
Success!!!! 

My initial setup was using both Ardour and Reaper on Ubuntu with my Focusrite 2i2 without any Focusrite drivers installed. I don't think there's an official Linux driver for the 2i2.  Part of my problem may have been that if I turned up the input from the amp past a certain level I was getting a low frequency feedback. That might have been the way Ubuntu routed audio to/from the interface.

Here's what I did to resolve the issues (some of these may or may not have had an impact):

1. I decided to boot into the Windows partition of my PC to run the capture
2. I downloaded/installed Reaper
3. I installed the drivers for my Focusrite 2i2. As soon as I did that it indicated there was an update so I installed the updated firmware.  Prior to the update (both on Ubuntu and Windows) I was getting some white noise on the input and after the update that stopped. Additionally, the low frequency feedback also stopped on Windows so I'm going to attribute that to using the official Focusrite driver.
4. I recorded the output of my amp with Reaper on Windows and exported the WAV file, being sure to not add the trailing "tail". I exported with 48kHz/24 bit/mono.
5. Somehow I was using the outdated training colab page. Not sure how I did that, may have been saved in my browsing history.  Here's the link that I first landed on: https://colab.research.google.com/github/sdatkinson/neural-amp-modeler/blob/bd1d8a5/bin/train/easy_colab.ipynb#scrollTo=xOzo3NxqjZjl. When I switched over to Windows I had to redownload the V3 wav and saw that the colab page that I got from the NAM documentation page was different.
6. The ESR was .0027124 and training completed!
